fastplotlib/tools/_histogram_lut.py

```diff
@@ -5,6 +5,7 @@
 
 import pygfx
 
+from ..utils.functions import subsample_array
```

Comment:
```suggestion
from ..utils import subsample_array
```

functions is in `utils.__all__`